invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/index.js

```diff
@@ -133,13 +139,10 @@ const LogoUploader = (props) => {
   };
 
   let deleteLogo = async () => {
-    const client = new CommunitiesApiClient();
-    const response = await client.deleteLogo(props.community.id);
-    if (response.code >= 400) {
-      props.onError(response.errors);
-    }
-  };
+    const client = new CommunityApi();
 
+    await client.deleteLogo(props.community.id);
```

Comment:
try catch missing + error handling

Reply:
it is included inside the `<DeleteButton/>`